asreviewcontrib/insights/metrics.py

```diff
@@ -187,7 +247,12 @@ def get_metrics(state_obj,
                 "id": "td",
                 "title": "Time to discovery",
                 "value": td
-            }]
+            }, {
+                "id": "confusion_matrix",
+                "title": "TPs, FPs, TNs, FNs",
```

Comment:
I think it might make sense to split the confusion matrix intro into 4 separate items. This avoids positional selections and makes it easier to flatten to a table (won't break Makita pipelines). You can also make use of proper titles. 

```python
my_fp_i = my_fp[i]
```
versus 
```python
my_fp_i = my_confusion_matrix[i][1]
```

This involves more command line options as well but makes it also easier to explain there.